assets/src/edit-story/utils/useIntersectionEffect.js

```diff
@@ -30,11 +30,7 @@ function useIntersectionEffect(ref, options = {}, handler, deps = undefined) {
   useEffect(
     () => {
       const node = ref.current;
```

Comment:
CMIIW but `options.root` will always be `null` here, so while fetching next page works (root=window), correct root is not taken into account. Generally I think we should use ref here in the past, [we already talked](https://github.com/google/web-stories-wp/pull/562#discussion_r393154492) about this.

Reply:
PTAL, I took a suggestion from https://github.com/facebook/react/issues/13029. It's not ideal because using a ref callback doesn't provide an easy way of passing that ref around, so I need to introduce another normal ref and set its "current" in the ref callback. Not super happy with this approach so happy to chat over slack about other suggestions you may have!

Reply:
Looks good. 
Extra `useRef` is not an issue for me, before that you had `container`, so it's the same in terms of LOC, of course we can have some kind of helper (like people do in #13029) - but not required since we don't have other places that would use it?